mealie/services/user_services/registration_service.py

```diff
@@ -70,6 +75,8 @@ def register_user(self, registration: CreateUserRegistration) -> PrivateUser:
         else:
             raise HTTPException(status.HTTP_400_BAD_REQUEST, {"message": "Missing group"})
 
+        self.logger.info(self.repos.users.get_by_username(registration.username))
+
```

Comment:
Did you mean to leave this in?

Reply:
No sorry it will be gone in the final PR